src/styles/card.scss

```diff
@@ -0,0 +1,115 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  font-family: Roboto, sans-serif;
+}
+
+.product-foto {
+  margin: 33px 20px 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.product {
+  margin: 0 17px;
+  box-sizing: border-box;
+
+  &-name {
+    width: 166px;
+    height: 36px;
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    color: #060b35;
+  }
+
+  &-code {
+    width: 98px;
+    height: 14px;
+    margin-bottom: 16px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+    color: #616070;
+  }
+}
+
+.rating {
+  margin: 0 16px 24px;
+  box-sizing: border-box;
+  display: flex;
+  gap: 17px;
+  height: 16px;
+  width: 166px;
+}
+
+.reviews {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  text-align: right;
+  color: #060b35;
+}
+
+.price {
+  margin: 0 16px 24px;
+  display: flex;
+  gap: 85px;
+
+  &-text {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    text-align: left;
+    color: #616070;
+  }
+
+  &-sum {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    text-align: right;
+    color: #060b35;
+  }
+}
+
+.buy {
+  box-sizing: border-box;
+  margin: 0 16px 16px;
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  opacity: 1;
+  background: #00acdc;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  &-link {
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-decoration: none;
+    color: #fff;
+  }
+
+  &:hover {
+    background: #f3f3f3;
+    border: 1px solid #00acdc;
+    color: #00acdc;
+  }
```

Comment:
The hover effect on the `.buy` class changes the background and border color, but it does not affect the text color of the `.buy-link`. To ensure the text color changes on hover, you should apply the color change to the `.buy-link` as well. Consider adding a rule for `.buy-link:hover` to change the text color.